src/ol/View.js

```diff
@@ -1237,6 +1237,21 @@ class View extends BaseObject {
    * @api
    */
   adjustResolution(ratio, opt_anchor) {
+    let anchor = opt_anchor;
+    if (anchor) {
+      anchor = fromUserCoordinate(opt_anchor, this.getProjection());
```

Comment:
How about:
```js
let anchor = opt_anchor && fromUserCoordinate(opt_anchor, this.getProjection());
```

Reply:
Done in 5d4e77151c3eb2c8cd509969a291f4d6c9e3402c.